src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import cl from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => selectedTabId === tab.id)
+    ?? tabs[0];
```

Comment:
You have the same code in the `App` component and there is an inconsistency
In the `App` you don't have fallback value, while here you do have it